packages/web-components/src/components/menu/menu-item-divider.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html } from 'lit';
+import { prefix } from '../../globals/settings';
+import styles from './menu-item.scss?lit';
+import { carbonElement as customElement } from '../../globals/decorators/carbon-element';
+/**
+ * Menu Item.
+ *
+ * @element cds-menu-item-divider
+ */
+@customElement(`${prefix}-menu-item-divider`)
+class CDSmenuItemDivider extends LitElement {
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'separator');
+    }
+    super.connectedCallback();
+  }
+  render() {
+    return html` <li></li> `;
+  }
```

Comment:
i believe this shouldn't be needed nowo and the styles can just be applied to the host element
```suggestion
```